connectors/csws/pom.xml

```diff
@@ -174,52 +174,44 @@
         </executions>
       </plugin>
 
-      <!--
+
+
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>build-helper-maven-plugin</artifactId>
+        <groupId>org.apache.cxf</groupId>
```

Comment:
usage of cxf-maven plugin is integration in the svn branch, but in version 3.2.6 (to be in sync with the version reference from the ant build)